demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -108,12 +108,10 @@ class NoModelException(Exception):
 
 
 class Neo4jContentGraphInterface(ContentGraphInterface):
-    # this is used to save cache of packs and integrations which queried
-    _import_handler = Neo4jImportHandler()
-
```

Comment:
This caused `Neo4jImportHandler()` to run whenever this module is imported, even if an instance of `Neo4jContentGraphInterface` was never initialized, creating the `neo4j` directory even on unrelated commands.

In this PR I updated `Neo4jImportHandler` to be a singleton, and then initialized `_import_handler` in `__init__` to resolve this issue.